manifests/m/M/C/vanderKooij/Picto-Selector/1.8.26/M.C.vanderKooij.Picto-Selector.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: M.C.vanderKooij.Picto-Selector

```

```suggestion
PackageIdentifier: MCvanderKooij.PictoSelector
```
Having the extra periods in the Identifier causes many sub-folders that aren't necessary